### PR TITLE
Fish/cleanup

### PR DESCRIPTION
**Pull request by discordianfish**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,12 +16,10 @@ push:
...
-	-it									\
-	--net=host								\
-	-v /vagrant/dcos-openvpn:/dcos						\
-	thomasr/dcos-openvpn							\
+	docker run -it --net=host       \
+	-v /vagrant/dcos-openvpn:/dcos  \
+	mesosphere/dcos-openvpn         \
 	/dcos/bin/test.sh
 
 setup-dns:
```
Comment (grampelberg) [inline Makefile:18]: Should probably just remove this, it was a half-thought solution to a problem that I'm not sure is a real problem (and it doesn't work at all right now).

File: marathon.json [hunk 2]
```diff
@@ -7,13 +7,13 @@
   "container": {
     "type": "DOCKER",
     "docker": {
-      "image": "thomasr/dcos-openvpn:25",
+      "image": "mesosphere/dcos-openvpn",
```
Comment (grampelberg) [inline marathon.json:10]: Need to update to force-pull as well.

File: bin/push.bash [hunk 3]
```diff
@@ -1,65 +0,0 @@
-#!/bin/bash -x
```
Comment (grampelberg) [inline bin/push.bash:1]: How does the auto-docker build stuff work? Is it just the dockerfile?

Comment (discordianfish) [inline bin/push.bash:1]: Yep, it's just running docker build. We can't have automatic tagging but we can also have a 'stable' tag or something.

File: marathon.json (hunk 2 above)
Comment (discordianfish) [inline marathon.json:10]: That's true, will submit that later - okay?

Comment (discordianfish) [inline marathon.json:10]: So the force-pull for the server container is already in master. I've just pushed another commit to force-pull the image for the framework as well.

**Pull request merged**